src/shared/actions/transfers.js

```diff
@@ -433,7 +433,10 @@ export const makeTransaction = (seedStore, receiveAddress, value, message, accou
     const withPreTransactionSecurityChecks = () => {
         // Progressbar step => (Validating receive address)
         dispatch(setNextStepAsActive());
-
+        //Check the last trit for validity
+        if(isLastTritZero(address) === false){
```

Comment:
`isLastTritZero` needs to be imported [here](https://github.com/7h7/trinity-wallet/blob/develop/src/shared/actions/transfers.js#L28) as well (it won't let me comment on that line):
```js
import {
    withRetriesOnDifferentNodes,
    fetchRemoteNodes,
    getRandomNodes,
    throwIfNodeNotHealthy,
    isLastTritZero,
} from '../libs/iota/utils';
```